src/Location/location.controller.ts

```diff
@@ -0,0 +1,36 @@
+import { Controller, Post, Body, Get, Param, Patch, Delete } from '@nestjs/common';
+import { LocationService } from './location.service';
+import { LocationDto } from './dto/location.dto';
+
+@Controller('location')
+export class LocationController {
+  constructor(private locationService: LocationService) {
```

Comment:
private members should start with `_` to indicate them as private and find them easily using auto complete